src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -82,5 +82,11 @@
   },
   "SearchAPI": {
     "SearchAPIEnabled": false
+  },
+  "CompilerHostAuth": {
```

Comment:
If I recall, there is a section for Compiler Host settings in this file. It will be nice to move these settings with that section so that all Compiler host settings are together.